Shared/Sources/RevertViewController.swift

```diff
@@ -63,6 +68,20 @@ class RevertTableViewController: UITableViewController, SettableHomeItem {
       self.prepareForInfoSegue(segue, item: self.item)
     }
   }
+
+  // MARK: Private
+
+  private static let overscanInsets = UIEdgeInsets(top: 60, left: 90, bottom: 60, right: 90)
+
+  private static func adjustTVInsetsForTableView(tableView: UITableView) {
+    tableView.layoutMargins = self.overscanInsets
+
+    // We also have to account for overscan on the top and the bottom of the table view itself.
+    // Content is initially offset so the table view doesn't jump when the inset update takes place.
+    tableView.contentInset.top = self.overscanInsets.top
+    tableView.contentInset.bottom = self.overscanInsets.bottom
+    tableView.contentOffset.y = -self.overscanInsets.top
+  }
```

Comment:
I think this method belongs on the tableview, not the view controller. Insets can also be defined there if they aren't view controller specific.